### DISCUSSION TITLE
Need search and tags in the site

### DISCUSSION THREAD

**Discussion by Hijongpark** [nesbox/TIC-80#1714] [Ideas]
Why there's no search and tag to find the games in the official site ? Many game sites and even pico8 has a search function to find the games I want to play.

**Comment by nesbox** [comment]
Hi,

we have open issue # 1575 but not yet implemented

**Comment by Chaiavi** [comment]
Will also love to see this feature
I think a "Search" function is not too complicated to implement, no ?

**Comment by matdombrock** [comment]
I would love to see this
